### ISSUE TITLE
Fix Deprecation warning for CRFEntityExtractor

### ISSUE THREAD

**Issue by akelad** [RasaHQ/rasa#5384]
```
/Users/akela/Documents/rasa-new/rasa/utils/common.py:347: UserWarning: 'CRFEntityExtractor' is deprecated and will be removed in version 2.0. Use 'DIETClassifier' instead.
  More info at https://rasa.com/docs/rasa/migration-guide/
```
Right now it's a `UserWarning`, I would guess it needs to be a `FutureWarning`. Also, can we link to the relevant section of the migration guide ?

**Comment by akelad** [comment]
Actually wait, @tabergma did we already undeprecate this?

**Comment by tabergma** [comment]
Yes. Will be released with 1.8.1.

**Comment by akelad** [comment]
ok closing.
